src/item.rb

```diff
@@ -0,0 +1,25 @@
+require 'date'
+
+class Item
+  attr_reader :id, :genre, :author, :source, :label, :publish_date, :archived
+
+  def initialize(genre, author, source, label, publish_date)
+    @id = rand(1..1000)
+    @genre = genre
+    @author = author
+    @source = source
+    @label = label
+    @publish_date = publish_date
+    @archived = false
+  end
+
+  def move_to_archive
+    @archived = true if can_be_archived?
+  end
+
+  private
+
+  def can_be_archived?
+    (Date.today.year - Date.parse(@publish_date).year) > 10
+  end
+end
```

Comment:
- [ ] [OPTIONAL] when entering an invalid date the user gets the below error, it might be nice if you catch this error so the application keeps running.

Reply:
Thank you for the recomendation, we will change it!